packages/primitives/popover/src/popover-content-attributes.component.ts

```diff
@@ -62,4 +69,9 @@ export class RdxPopoverContentAttributesComponent {
                 (this.popoverRoot.cssClosingAnimation() && this.popoverRoot.state() === RdxPopoverState.CLOSED))
         );
     }
+
+    /** @ignore */
+    ngOnInit() {
```

Comment:
it seems necessary to call `focusFirstTabbableElement` after component content inititialization 

```suggestion
    ngAfterContentInit() {
```

Reply:
What about `afterNextRender` function util? :)

Reply:
+ `afterNextRender` :)

Reply:
> @artembelik @pawel-twardziak Thanks a lot for the helpful feedback - it's really appreciated!
> 
> I thought `ngAfterContentInit` would be more appropriate. But the Contributing rules suggest avoiding such hooks. I tested `ngOnInit` and it works.
> 
> The suggestion to use `afterNextRender` solves both of these problems. It can be used in the constructor. I need to decide on which phase to use it.

https://angular.dev/api/core/afterNextRender - important thing is that `afterNextRender` has a few types of callback: 
Callbacks run in the following phase order once, after the next render:
-     earlyRead
-     write
-     mixedReadWrite
-     read
Remember to pick the right one :)